stanford/roles/cluster/defaults/main.yml

```diff
@@ -188,6 +206,32 @@ CLUSTER_SECURITY_GROUP_RULES:
       to_port: 80
       cidr_ip: '0.0.0.0/0'
 CLUSTER_ELBS:
+  analytics_jenkins:
+    - name: "{{ COMMON_DEPLOYMENT }}-elb-analytics-jenkins-public"
+      health_check:
+        ping_protocol: http
+        ping_path: '/health/'
```

Comment:
Are we using this on CME?
I didn't realize this endpoint existed...

Reply:
yes.

Reply:
Per f2f, change to `cc.xml`; see `build` cluster as reference.